data/scripts/ocd2geojson.py

```diff
@@ -0,0 +1,22 @@
+import sys
+import json
+
+ocd_map = json.load(sys.stdin)
+geojson = {'type': "FeatureCollection",
+            'features': []}
+
+name = sys.argv[1]
+
+try:
```

Comment:
This could use some additional context, namely: What are the potential shapes of the data from the OCD API? Why is there more than one?

Reply:
i don't think we'll re-use this code except to create boundary.geojson this one time. 

If boundaries change, we'll not stuff them into ocd the API got produce boundary.geojson. 

What about just adding a README in the data directory that gives this context.

Reply:
Sounds good to me!